src/main/java/jimmy/mcgymmy/model/McGymmy.java

```diff
@@ -83,6 +83,15 @@ public void removeFood(Index index) {
         foodItems.remove(index);
     }
 
+    /**
+     * Removes {@code toRemove} from this {@code McGymmy}.
+     * {@code toRemove} must be valid.
+     */
+    public void removeFood(Food toRemove) {
```

Comment:
I think with the new implementation we can delete this method and the other related one in `Fridge`